packages/account-management/src/routes/index.ts

```diff
@@ -5,8 +5,7 @@ const unAuthenticatedPages: string[] = ['/register', '/validated', '/error'];
 export const indexPage: RouteMiddleware = (context) => {
   const bundle = context.routes.url('assets-bundles');
 
-  const isUnauthenticatedPage: boolean = !!context.req.url && unAuthenticatedPages.includes(context.req.url.split('?')[0]);
-  if (!isUnauthenticatedPage && !context.isAuthenticated()) {
+  if (!unAuthenticatedPages.includes(context.request.URL.pathname) && !context.isAuthenticated()) {
```

Comment:
Sorry, this just came to me now that we've made the condition clearer. Would it be better to flip the logic?

```ts
  if (context.isAuthenticated() || unAuthenticatedPages.includes(context.request.URL.pathname)) {
    context.response.body = '<html />';
    return;
  }

  context.redirect('/login');
```

Reply:
Yeah, I think that's a little better. Can get rid of the the `not unauthenticatedPage` double negative and make the redirect to login the default condition that way